examples/chainermn/cifar/train_cifar.py

```diff
@@ -0,0 +1,138 @@
+    parser.add_argument('--resume', '-r', default='',
+                        help='Resume the training from snapshot')
+    parser.add_argument('--early-stopping', type=str,
+                        help='Metric to watch for early stopping')
+    args = parser.parse_args()
+
+    # Prepare ChainerMN communicator.
+    if args.gpu:
+        comm = chainermn.create_communicator('hierarchical')
+        data_axis, model_axis = comm.rank % 2, comm.rank // 2
+        data_comm = comm.split(data_axis, comm.rank)
+        model_comm = comm.split(model_axis, comm.rank)
+        device = comm.intra_rank
+    else:
+        comm = chainermn.create_communicator('naive')
+        data_axis, model_axis = comm.rank % 2, comm.rank // 2
+        data_comm = comm.split(data_axis, comm.rank)
+        model_comm = comm.split(model_axis, comm.rank)
+        device = -1
+
+    if model_comm.size != 2:
+        raise ValueError(
+            'This example can only be executed on the even number'
+            'of processes.')
+
+    
+    if comm.rank == 0:
+        print('==========================================')
+        print('Num process (COMM_WORLD): {}'.format(comm.size))
+        if args.gpu:
+            print('Using GPUs')
+        print('Num Minibatch-size: {}'.format(args.batchsize))
+        print('Num epoch: {}'.format(args.epoch))
+        print('==========================================')
+
+    
+    # Set up a neural network to train.
+    # Classifier reports softmax cross entropy loss and accuracy at every
+    # iteration, which will be used by the PrintReport extension below.
+    if model_axis == 0:
+        if args.dataset == 'cifar10':
+            print('Using CIFAR10 dataset.')
+            class_labels = 10
+            train, test = get_cifar10()
+        elif args.dataset == 'cifar100':
+            print('Using CIFAR100 dataset.')
+            class_labels = 100
+            train, test = get_cifar100()
+        else:
+            raise RuntimeError('Invalid dataset choice.')
+    model = L.Classifier(models.VGG.VGG(class_labels))
+    
+    if device >= 0:
+        # Make a specified GPU current
+        chainer.backends.cuda.get_device_from_id(device).use()
+        model.to_gpu()  # Copy the model to the GPU
+
+    optimizer =  chainermn.create_multi_node_optimizer(chainer.optimizers.MomentumSGD(args.learnrate), data_comm)
+    optimizer.setup(model)
+    optimizer.add_hook(chainer.optimizer_hooks.WeightDecay(5e-4))
+
+    
+    train = chainermn.scatter_dataset(train, data_comm, shuffle = True)
+    test = chainermn.scatter_dataset(test, data_comm, shuffle = True)
+
+    train_iter = chainer.iterators.SerialIterator(train, args.batchsize, shuffle = False) 
+    test_iter = chainer.iterators.SerialIterator(test, args.batchsize,
+                                                 repeat=False, shuffle=False)
+
+    stop_trigger = (args.epoch, 'epoch')
+    # Early stopping option
+    if args.early_stopping:
```

Comment:
Please remove the early stopping stuff as mentioned above.

Reply:
remove it